Bump aws-sdk to 2.1005+

francoisferrand opened the pull request:
Use same spec as other packages (utapi, vault...), and allow automatic
version bump (dependabot).

Issue: ARSN-234

francoisferrand committed:
Bump aws-sdk to 2.1005+

Use same spec as other packages (utapi, vault...), and allow automatic
version bump (dependabot).

Issue: ARSN-234

francoisferrand commented:
/create_pull_requests

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/arsenal/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/8.1/improvement/ARSN-234` with contents from `improvement/ARSN-234`
and `development/8.1`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/8.1/improvement/ARSN-234 origin/development/8.1
 $ git merge origin/improvement/ARSN-234
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/8.1/improvement/ARSN-234
```





*The following options are set:* **create_pull_requests**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/ARSN-234` into
`development/7.10`
* [w/8.1/improvement/ARSN-234](https://github.com/scality/arsenal/compare/development/8.1...w/8.1/improvement/ARSN-234) will be merged into `development/8.1` (pull request #1943)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.







*The following options are set:* **create_pull_requests**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers











*The following options are set:* **create_pull_requests**

bert-e commented:
# History mismatch



Merge commit #6cdb93d37a0552672f883e8d5f8885e10adfe10e on the integration branch
`w/8.1/improvement/ARSN-234` is merging a branch which is neither the current
branch `improvement/ARSN-234` nor the development branch
`development/8.1`.

It is likely due to a rebase of the branch `improvement/ARSN-234` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **create_pull_requests**

francoisferrand committed:
Remove check with empty bucket name

This test is not relevant, since a bucket cannot have an empty name;
and there is now a check in AWS SDK which rejects the request directly.

Issue: ARSN-234

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/ARSN-234` into
`development/7.10`
* [w/8.1/improvement/ARSN-234](https://github.com/scality/arsenal/compare/development/8.1...w/8.1/improvement/ARSN-234) will be merged into `development/8.1` (pull request #1945)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.







*The following options are set:* **create_pull_requests**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers











*The following options are set:* **create_pull_requests**

francoisferrand commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/7.10`

* :heavy_check_mark: `development/8.1`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **create_pull_requests, approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/7.10`

* :heavy_check_mark: `development/8.1`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.4`



Please check the status of the associated issue ARSN-234.

Goodbye francoisferrand.
